packages/engine-server/src/markdown/decorations/hashTags.ts

```diff
@@ -46,7 +47,10 @@ export async function decorateTag({
     fname,
     engine,
   });
-  if (colorType === "configured" || !engine.config.noRandomlyColoredTags) {
+  if (
+    colorType === "configured" ||
+    !DConfig.readConfigSync(engine.wsRoot).noRandomlyColoredTags
```

Comment:
this will be called a whole bunch of times

Reply:
Got it. I'll pass the config down from the engine so it doesn't need to be called